### PR TITLE
fix No database duplicate.

### LINKED ISSUE #8243: No database - Text duplicate in the generator step.
##### **Overview of the issue**

Is a simple visual bug, in the process to generate microservice show "No database" 2 times.

##### **Motivation for or Use Case**

Create new microservice.

##### **Reproduce the error**

run jhipster

WARNING! Your Node version is not LTS (Long Term Support), use it at your own risk! JHipster does not support non-LTS releases, so if you encounter a bug, please use a LTS version first.
? May JHipster anonymously report usage statistics to improve the tool over time? No
? Which *type* of application would you like to create? Microservice application
? What is the base name of your application? myApp
? As you are running in a microservice architecture, on which port would like your server to run? It should be unique to avoid port conflicts. 8081
? What is your default Java package name? com.mycompany.myapp
? Which service discovery server do you want to use? No service discovery
? Which *type* of authentication would you like to use? JWT authentication (stateless, with a token)
? Which *type* of database would you like to use? 
  SQL (H2, MySQL, MariaDB, PostgreSQL, Oracle, MSSQL) 
❯ MongoDB 
  Couchbase 
  No database 
  No database 
  Cassandra 

##### **Related issues**

No

##### **Suggest a Fix**

N/A

##### **JHipster Version(s)**

5.3.1

##### **JHipster configuration**

Using JHipster version installed locally in current project's node_modules
Executing jhipster:info
Options: 
Welcome to the JHipster Information Sub-Generator

##### **JHipster Version(s)**

```
my-app@0.0.0 /usr/home/developer/Documents/Project/poc/myApp
└── generator-jhipster@5.3.1 

```

##### **JHipster configuration, a `.yo-rc.json` file generated in the root folder**

<details>
<summary>.yo-rc.json file</summary>
<pre>
{
  "generator-jhipster": {
    "promptValues": {
      "packageName": "com.mycompany.myapp"
    },
    "jhipsterVersion": "5.3.1",
    "applicationType": "microservice",
    "baseName": "myApp",
    "packageName": "com.mycompany.myapp",
    "packageFolder": "com/mycompany/myapp",
    "serverPort": "8081",
    "authenticationType": "jwt",
    "cacheProvider": "no",
    "enableHibernateCache": false,
    "websocket": false,
    "databaseType": "mongodb",
    "devDatabaseType": "mongodb",
    "prodDatabaseType": "mongodb",
    "searchEngine": false,
    "messageBroker": false,
    "serviceDiscoveryType": false,
    "buildTool": "maven",
    "enableSwaggerCodegen": false,
    "jwtSecretKey": "bXktc2VjcmV0LXRva2VuLXRvLWNoYW5nZS1pbi1wcm9kdWN0aW9uLWFuZC10by1rZWVwLWluLWEtc2VjdXJlLXBsYWNl",
    "enableTranslation": false,
    "testFrameworks": [
      "gatling"
    ],
    "jhiPrefix": "jhi",
    "clientPackageManager": "npm",
    "skipClient": true,
    "skipUserManagement": true
  }
}
</pre>
</details>

##### **JDL for the Entity configuration(s) `entityName.json` files generated in the `.jhipster` directory**

<details>
<summary>JDL entity definitions</summary>

<pre>

</pre>
</details>

##### **Environment and Tools**

openjdk version "1.8.0_162"
OpenJDK Runtime Environment (build 1.8.0_162-b12)
OpenJDK 64-Bit Server VM (build 25.162-b12, mixed mode)

git version 2.16.2

node: v9.5.0

npm: 5.6.0

yarn: 1.3.2

Docker version unknown-version, build unknown-commit

##### **Entity configuration(s) `entityName.json` files generated in the `.jhipster` directory**

N/A

##### **Browsers and Operating System**

N/A I used postman or curl for microservcies

- [x] Checking this box is mandatory (this is just to show you read everything)

### PR DESCRIPTION
**Pull request by tomj0101**
fix No database - Text duplicate in the generator step Fix #8243.
I did my internal test and work well!

- Please make sure the below checklist is followed for Pull Requests.

- [x] [Travis tests](https://travis-ci.org/jhipster/generator-jhipster/pull_requests) are green
- [x] Tests are added where necessary
- [x] Documentation is added/updated where necessary
- [ ] Coding Rules & Commit Guidelines as per our [CONTRIBUTING.md document](https://github.com/jhipster/generator-jhipster/blob/master/CONTRIBUTING.md) are followed

Fix #8243

### CODE REVIEW

File: generators/server/prompts.js [hunk 1]
```diff
@@ -182,14 +182,6 @@ function askForServerSideOpts(meta) {
...
                     }
-                    if (
-                        (response.authenticationType !== 'oauth2' && applicationType === 'microservice')
-                        || (response.authenticationType === 'uaa' && applicationType === 'gateway')
-                    ) {
-                        opts.push({
-                            value: 'no',
-                            name: 'No database'
-                        });
                     }
```
Comment (pascalgrimaud) [inline generators/server/prompts.js]: I think you need to include this additional `}` in the deleted part too

Comment (tomj0101) [reply generators/server/prompts.js]: sure!

Comment (pascalgrimaud) [reply generators/server/prompts.js]: sorry, I wasn't clear enough: you need to delete `}`, not add a new one. You can check your code with npm test or yarn test

Comment (pascalgrimaud) [reply generators/server/prompts.js]: right, travis will test your last commit

**Comment by tomj0101** [comment]
Hey @pascalgrimaud I go to sleep but I will keep my eyes in this after wake up, I am in New_York/Timezone!

**Comment by pascalgrimaud** [comment]
thanks and good night
It will be merged as soon as Travis passed

**Pull request merged**